.nvmrc

```diff
@@ -1 +1 @@
-10.22.0
+18.18.0
```

Comment:
:bulb: let's use the `lts/hydrogen` tag here instead of being so specific

```suggestion
lts/hydrogen
```

Reply:
Good, I will apply the suggestion